src/Commands/Local.hs

```diff
@@ -36,15 +36,17 @@ localCommand e args = do
       bss <- mapM LB.readFile fs
       res <- runExceptT $ do
         allTxs <- hoistEither $ first unlines $ parseAsJsonOrYaml bss
-        hpPairs <- handleOptionalNode e allTxs $ _nodeTxCmdArgs_node args
-        forM hpPairs $ \(hp, txs) -> do
-          n <- ExceptT $ getNode le hp
+        shpPairs <- handleOptionalNode e allTxs $ _nodeTxCmdArgs_node args
+        forM shpPairs $ \(shp, txs) -> do
+          logEnv e DebugS $ logStr "Here A"
```

Comment:
I assume this were for debugging and should be removed now?

Reply:
Ahh yes.  Will update.